chpl/chpl-resources/src/main/resources-production/jobs.xml

```diff
@@ -12,127 +12,45 @@
-                <cron-expression>0 0 3 * * ?</cron-expression> <!-- At 0300 UTC every day -->
-            </cron>
-        </trigger>
-        <!--  update inheritance errors report to run at 1030 -->
-        <trigger>
-            <cron>
-                <name>generateInheritanceErrorsReport</name>
-                <group>inheritanceErrorsReportTrigger</group>
-                <job-name>inheritanceErrorsReportCreator</job-name>
-                <job-group>systemJobs</job-group>
-                <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
-                <cron-expression>0 30 10 * * ?</cron-expression> <!-- At 1030 UTC every day -->
-            </cron>
-        </trigger>
-        <!--  update broken surv rules report to run at 0300 UTC -->
-        <trigger>
-            <cron>
-                <name>generateBrokenSurveillanceRules</name>
-                <group>brokenSurveillanceRulesTrigger</group>
-                <job-name>brokenSurveillanceRulesCreator</job-name>
-                <job-group>systemJobs</job-group>
-                <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
-                <cron-expression>0 0 2 * * ?</cron-expression> <!-- At 0200 UTC every day -->
-            </cron>
-        </trigger>
-        <!--  update chart data job to run at 0110 UTC -->
-        <trigger>
-            <cron>
-                <name>generateChartData</name>
-                <group>chartDataCreatorTrigger</group>
-                <job-name>chartDataCreator</job-name>
-                <job-group>systemJobs</job-group>
-                <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
-                <cron-expression>0 10 01 * * ?</cron-expression> <!-- At 0110 UTC every day -->
-            </cron>
-        </trigger>
-        <!--  update API key warning job to run at 0100 UTC -->
-        <trigger>
-            <cron>
-                <name>apiKeyDeleteWarningEmail</name>
-                <group>apiKeyDeleteWarningEmailTrigger</group>
-                <job-name>apiKeyDeleteWarningEmailJob</job-name>
-                <job-group>systemJobs</job-group>
-                <misfire-instruction>MISFIRE_INSTRUCTION_FIRE_ONCE_NOW</misfire-instruction>
-                <cron-expression>0 0 1 1/1 * ? *</cron-expression> <!-- At 0100 UTC every day -->
-            </cron>
-        </trigger>
-        <!--  update API Key delete job to run at 0105 UTC -->
-        <trigger>
-            <cron>
-                <name>apiKeyDeleteEmail</name>
-                <group>apiKeyDeleteTrigger</group>
-                <job-name>apiKeyDeleteJob</job-name>
-                <job-group>systemJobs</job-group>
-                <misfire-instruction>MISFIRE_INSTRUCTION_FIRE_ONCE_NOW</misfire-instruction>
-                <cron-expression>0 05 1 1/1 * ? *</cron-expression> <!-- At 0105 UTC every day -->
-            </cron>
-        </trigger>
+        <!-- Broken URL Data Collector -->
```

Comment:
Should add a checklist item to this ticket to remove this section after PROD push